examples/analysis/point_extraction/interpolation.py

```diff
@@ -0,0 +1,56 @@
+"""
+Regular-grid interpolation
+==========================
+
+This example demonstrates raster interpolation to point values using :func:`~geoutils.Raster.interp_points`.
```

Comment:
This example demonstrates the extraction of raster values at any given location, using interpolation.

Reply:
Clarified